src/langs/en/uppy.json

```diff
@@ -0,0 +1,64 @@
+{
+  "closeModal": "Close Modal",
+  "addMoreFiles": "Add more files",
+  "addingMoreFiles": "Adding more files",
+  "importFrom": "Import from %{name}",
```

Comment:
Is this file used by uppy directly ? So this is why the template string is different ?

Reply:
Exactly!